futures-core/src/future/result.rs

```diff
@@ -29,7 +29,7 @@ impl<T, E> IntoFuture for result::Result<T, E> {
 /// # Examples
 ///
 /// ```
-/// use futures::future::*;
+/// use futures_core::future::*;
```

Comment:
One thing to think about: these doc tests will show up in the crate docs for `futures`, so people using `futures` directly might be confused when they see references to the individual crates in the docs.